diff --git a/emscripten/tools/shared.orig.py b/emscripten/tools/shared.py
index 3f05858..d202814 100644
--- a/emscripten/tools/shared.orig.py
+++ b/emscripten/tools/shared.py
@@ -432,14 +432,14 @@ def check_sanity(force=False, quiet=False):
     if sanity_is_correct():
       return

-    if os.path.exists(sanity_file):
-      sanity_data = utils.read_file(sanity_file)
-      logger.info('old sanity: %s', sanity_data.strip())
-      logger.info('new sanity: %s', expected.strip())
-      logger.info('(Emscripten: config changed, clearing cache)')
-      cache.erase()
-    else:
-      logger.debug(f'sanity file not found: {sanity_file}')
+    #if os.path.exists(sanity_file):
+    #  sanity_data = utils.read_file(sanity_file)
+    #  logger.info('old sanity: %s', sanity_data.strip())
+    #  logger.info('new sanity: %s', expected.strip())
+    #  logger.info('(Emscripten: config changed, clearing cache)')
+    #  cache.erase()
+    #else:
+    #  logger.debug(f'sanity file not found: {sanity_file}')

     perform_sanity_checks()

